scenario_groups.yml

```diff
@@ -77,3 +77,6 @@ DEBUGGER_SCENARIOS:
   - DEBUGGER_MIX_LOG_PROBE
   - DEBUGGER_PII_REDACTION
   - DEBUGGER_EXPRESSION_LANGUAGE
+
+ANDREW:
```

Comment:
Maybe a more descriptive group name would be more suitable?

Reply:
+1

Reply:
oops! I don't think I actually need this scenario group at all? This was from my time trying to figure out / use scenario groups :P I've removed this